src/nginx_conf_gen.sh

```diff
@@ -14,31 +13,32 @@ function generate_nginx_conf() {
   if [[ "${TRAVIS_BRANCH}" = "master" ||  "${TRAVIS_BRANCH}" = "main" || "${TRAVIS_BRANCH}" =~ "beta" ]]; then
     PREFIX="/beta"
   fi
-
+  
   echo "server {
-   listen 8000;
-   server_name $APP_NAME;
+    listen 8000;
+    server_name $APP_NAME;
+    error_log stderr;
 
-   location / {
-    try_files \$uri \$uri/ $PREFIX/apps/chrome/index.html;
-   }
+    location / {
+      try_files \$uri \$uri/ $PREFIX/apps/chrome/index.html;
+    }
 
-   location $PREFIX/apps/$APP_NAME {
-     alias /opt/app-root/src;
-   }
+    location $PREFIX/apps/$APP_NAME {
+      alias /opt/app-root/src/dist;
+    }
```

Comment:
This will not work unfortunately as this config is used from the `dist` folder as well. However we can change completely the rule to look like
```suggestion
    location ~* ^$PREFIX/apps/$APP_NAME/(.*) {
      alias /opt/app-root/src/;
      try_files /\$1 /dist/\$1 /dist/\$1;
    }
```
I'd try running this generate script and see if the config looks like:
```
 location ~* ^/apps/inventory/(.*) {
   alias /opt/app-root/src/;
   try_files /$1 /dist/$1 /dist/$1;
 }
```